src/main/java/org/terasology/launcher/settings/Settings.java

```diff
@@ -0,0 +1,54 @@
+package org.terasology.launcher.settings;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Properties;
+
+//TODO: should this be called `SettingsController` and also carry out some UI handling, e.g., displaying error messages
+//      to the user?
+public class Settings {
```

Comment:
This class is interesting as I moved the IO logic here. I'm wondering whether this can/should be part of the responsibility of `SettingsController` itself, or whether it should stay some additional helper class to be used by the controller class... :thinking: